build/build.mk

```diff
@@ -41,15 +43,33 @@ deps:
 	@$(GO_BIN) mod vendor
 	@echo '[go-get] Done.'
 
-.PHONY: test-coverage
-test-coverage: TEST_FLAGS += -covermode=atomic -coverprofile=$(COVERAGE_FILE)
-test-coverage: deps
+.PHONY: unit-test-with-coverage
+unit-test-with-coverage: TEST_FLAGS += -covermode=atomic -coverprofile=$(COVERAGE_FILE)
+unit-test-with-coverage: deps
 	@printf '\n================================================================\n'
 	@printf 'Target: test-coverage'
 	@printf '\n================================================================\n'
 	@echo '[test] Testing packages: $(SOURCE_FILES)'
 	$(GO_BIN) $(GO_TEST)
 
+#.PHONY: slow-unit-tests
+#test-coverage-with-slow: deps
+#	@printf '\n================================================================\n'
+#	@printf 'Target: test-slow'
+#	@printf '\n================================================================\n'
+#	@echo '[test] Testing packages: $(SOURCE_FILES)'
+#	$(GO_BIN) $(GO_TEST_WITH_SLOW)
+#	@echo '[test] Converting: $(COVERAGE_FILE) into lcov.info'
+#	@(gcov2lcov -infile=$(COVERAGE_FILE) -outfile=lcov.info)
+
```

Comment:
☝️ Is it necessary to maintain this commented or can we remove it? If we need to maintain I would fix the name

Reply:
Good point, I will clean this